apps/src/labs/labRedux.ts

```diff
@@ -97,10 +97,7 @@ export const setUpWithLevel = createAsyncThunk(
       levelProperties.isProjectLevel,
       false
     );
-    const disableProjects = convertOptionalStringToBoolean(
-      levelProperties.disableProjects,
-      false
-    );
+    const disableProjects = !!levelProperties.disableProjects;
```

Comment:
Does this still work correctly if the boolean is stringified on the backend ("true")?

Reply:
I tested end-to-end and the boolean isn't stringified in this flow.  Would we expect a string to be used instead?

Reply:
Hmm, I'm not sure - I'm noticing that on some levels (for example "New Music Lab Project.level"), boolean values are stored as strings (ex. `"is_project_level": "true"`). Maybe a side effect of properties being serialized from levelbuilder? We might need to update `convertOptionalStringToBoolean` to account booleans as well as strings.

Reply:
I've updated that function to handle both types now.